test/integration/accounts.ts

```diff
@@ -31,7 +31,7 @@ import {
 } from '../../src/utils/errors'
 
 describe('Accounts', function () {
-  let aeSdk, aeSdkWithoutAccount
+  let aeSdk: any, aeSdkWithoutAccount: any
```

Comment:
Todo: replace any with type once all stamps are removed and we have a global class/type